configure.ac

```diff
@@ -350,13 +343,6 @@ AC_ARG_ENABLE(suse,
 	]
 )
 
-AC_ARG_ENABLE(suse-sysv,
-	[  --enable-suse-sysv      obsoleted ],[
-	echo "ERROR: --enable-suse-sysv is obsoleted. Use --enable-suse."
-	exit 1
-	]
-)
-
```

Comment:
The `--enable-suse` argument needs to be removed as well.

Reply:
Done